data-hub/src/hub-in-a-box/root/com.marklogic.hub/lib/flow-lib.xqy

```diff
@@ -274,7 +409,14 @@ declare function flow:run-flow(
     ()
 };
 
+(:~
+ : Construct an envelope
+ :
+ : @param $map - a map with all the stuff in it
+ : @return - the newly constructed envelope
+ :)
 declare function flow:make-envelope($map as map:map)
```

Comment:
Envelope is one of your transforms, right? I get that you're providing a built-in one, but would it be helpful to put this in a separate file to illustrate how to build other ones?